packages/thumbprint-global-css/package.json

```diff
@@ -26,7 +26,8 @@
         "autoprefixer": "^9.0.1",
         "cssnano": "^4.1.0",
         "node-sass-tilde-importer": "2.0.0-alpha.1",
-        "postcss-cli": "^6.1.1",
+        "postcss": "^8.4.12",
```

Comment:
Why is this needed to make it work? Does the CLI have a peerDep?

Reply:
it's not even specified as a peerDeep -- the CLI just errors if you run without it.